runtime/v2/shim/shim.go

```diff
@@ -92,7 +95,6 @@ func parseFlags() {
 	flag.StringVar(&bundlePath, "bundle", "", "path to the bundle if not workdir")
 
 	flag.StringVar(&addressFlag, "address", "", "grpc address back to main containerd")
-	flag.StringVar(&containerdBinaryFlag, "publish-binary", "containerd", "path to publish binary (used for publishing events)")
```

Comment:
Is this considered a breaking change? Or is it ok to say that you cant use a "newer shim" with an "older containerd".

Reply:
I'm still provided the path on the containerd side so older compiled shims will still work

Reply:
Oh I gotchya you are right.